tests/Cucumber/CompatibilityTest.php

```diff
@@ -171,6 +170,13 @@ private function normaliseFeature(?FeatureNode $featureNode): ?FeatureNode
             $this->setPrivateProperty($featureNode, 'description', $trimmedDescription);
         }
 
+        if ($featureNode->getBackground()) {
+            foreach ($featureNode->getBackground()->getSteps() as $step) {
+                $this->setPrivateProperty($step, 'keywordType', '');
+                $this->setPrivateProperty($step, 'arguments', []);
+            }
+        }
+
```

Comment:
We need to do the same fixes to be able to compare steps in a Background that we already do for steps in a Scenario (for now, at least).

Reply:
this will conflict with #319 which changes this normalization logic. As the other PR is approved, I suggest merging it before finishing the work here.

Reply:
Agreed, I'd meant to note that in my annotation.

Reply:
FYI #319 is not merged yet because it's so far only approved by me and per our policy we want 2 approvals on PRs that are not from a maintainer.